liota/dccs/iotcc.py

```diff
@@ -177,9 +177,8 @@ def create_relationship(self, reg_entity_parent, reg_entity_child):
             Parameters:
             - obj: The object that has just obtained an UUID
         """
-        # sanity check: must be RegisteredEntity or RegisteredMetricRegisteredMetric
-        if (not isinstance(reg_entity_parent, RegisteredEntity) \
-                    and not isinstance(reg_entity_parent, RegisteredMetric)) \
+        # sanity check: must be RegisteredEntity or RegisteredMetric
+        if (not isinstance(reg_entity_parent, RegisteredEntity)) \
```

Comment:
This doesn't look like it'll parse correctly? if (()) will close the statement and should be a syntax error on the next line?

Reply:
In python, if statement is parsed till a matching colon (:) is received.

Reply:
Thank you! I missed that when I tried to check the python docs.